app/lib/brexit_checker/validators/group_validator.rb

```diff
@@ -0,0 +1,24 @@
+class BrexitChecker::Validators::GroupValidator < ActiveModel::Validator
+  CITIZEN_KEYS = %w[ visiting-eu
+                     visiting-uk
+                     visiting-ie
+                     living-eu
+                     living-ie
+                     living-uk
+                     working-uk
+                     studying-eu
+                     studying-uk
+                     common-travel-area ].freeze
+
+  def validate(record)
+    validate_citizen_group(record)
+  end
+
+private
+
+  def validate_citizen_group(record)
+    unless CITIZEN_KEYS.include? record.key
```

Comment:
Interesting how does this line get past rubocop?
I thought rubocop enforces parenthesis...
so would be:
```
    unless CITIZEN_KEYS.include?(record.key)
```

Surprised the linter doesn't get stroppy?

Reply:
Yes that is weird based on (the docs)[https://rubystyle.guide/#method-invocation-parens]. I grepped finder frontend and we seems to do it both ways. I'll parenthesise it though, thanks.

Reply:
Ah found an unrelated [ruby style guide example](https://rubystyle.guide/#ranges-or-between) where `.include?` is used, and they parenthesise 👍